components/foget-password.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image';
+import layoutTop from '@/assets/images/LayoutTop.png'
+import MarketingForget from '@/assets/images/Marketing-forget-password.png'
```

Comment:
I would like to suggest you to rename the image import name into camelcase, i.e. use `marketingForget` instead of `MarketingForget`